typify-impl/src/type_entry.rs

```diff
@@ -1527,10 +1527,14 @@ impl TypeEntry {
                         }
                     }
                 });
+
                 let pat = pattern.as_ref().map(|p| {
                     let err = format!("doesn't match pattern \"{}\"", p);
                     quote! {
-                        if regress::Regex::new(#p).unwrap().find(value).is_none() {
+                        static PATTERN: std::sync::LazyLock<regress::Regex> = std::sync::LazyLock::new(|| {
```

Comment:
```suggestion
                        static PATTERN: ::std::sync::LazyLock<::regress::Regex> = ::std::sync::LazyLock::new(|| {
```

I've been trying to prefix everything with `::` -- apparently I had missed this one in regress.

Reply:
will fix